server/src/main/java/com/lunchtime/service/MenuItemDishService.java

```diff
@@ -14,4 +14,7 @@
     Optional<MenuItemDish> findById(Long id);
 
     Page<MenuItemDish> findByRestaurantId(Long id, Pageable pageable);
+
+    Page<MenuItemDish> findDishesByRestaurantIdAndDishCategoryName(String name,
```

Comment:
I think 3rd method is quite long...
dishesByRestaurantId is redundant because it's clear that we are searching dishes in specific rest.
maybe "findByCategory" or "findByRestCategory"

Reply:
The name may be long, but other developers immediately understand what the method does. I suggest shortening the name to "findDishesByRestaurantAndCategory"

Reply:
> dishesByRestaurantId is redundant because it's clear that we are searching dishes in specific rest.

It isn't clear. There is parameter `id` in this method, it isn't explain what it is. 
`id` - `restaurantId` or `dishId`, or `categoryId`?

Reply:
Thank you both, I have changed the name to "findDishesByRestaurantIdAndCategoryName". I will push changes in few minutes.